src/Workspaces/Core/Portable/Experiments/IExperimentationService.cs

```diff
@@ -38,5 +38,6 @@ internal static class WellKnownExperimentNames
         public const string RemoveUnusedReferences = "Roslyn.RemoveUnusedReferences";
         public const string LSPCompletion = "Roslyn.LSP.Completion";
         public const string UnnamedSymbolCompletionDisabled = "Roslyn.UnnamedSymbolCompletionDisabled";
+        public const string RazorLspEditorFeatureFlag = "Razor.LSP.Editor";
```

Comment:
Do we have a tracking item for when they remove the feature flag we don't break?

Reply:
https://github.com/dotnet/aspnetcore/issues/31935